ui_framework/test/common_tests.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render } from 'enzyme';
+
+/**
+ * Ensures the rendered class matches the jest snapshot.
+ * @param reactClass
+ * @param props
+ */
+export function basicRenderTest(reactClass, props = {}) {
```

Comment:
I'm -1 on these functions. I prefer having assertions in the tests. I don't think this is very readable:

```
test('is rendered', () => basicRenderTest(KuiToolBarSearchBox, props));
```

But with this I instantly understand what's going on in the test:

```js
test('is rendered', () => {
  const component = <KuiToolBarSearchBox
    someProp='someValue' />
  expect(render(component)).toMatchSnapshot();
})
```

I think `basicRenderTest` is DRY-ing up something that ends up creating lots of one-off functions (like in this file), so it's actually less readable in the tests.

E.g. for the html attributes I'd do:

```js
test('html attributes', () => {
  const component = <KuiToolBarSearchBox
    someProp='someValue'
    { ...basicHtmlAttributes } />
  expect(render(component)).toMatchSnapshot();
})
```

and just import the html attributes if it's always the same attributes everywhere. Otherwise I'd just `const` them at the top of the relevant test file.